src/bin.rs

```diff
@@ -0,0 +1,18 @@
+#[macro_use]
```

Comment:
Do you still have use this? 

I think you can now just use `clap::clapp_app;`

Reply:
ah yes, good call. updated 646714d